docs/faq/gce.md

```diff
@@ -281,16 +281,38 @@ We plan to fix this [soon](https://github.com/kubernetes/kubernetes/issues/16337
 
 ## How do I disable the GCE Ingress controller?
 
-3 options:
-1. Have it no-op based on the `ingress.class` annotation as shown [here](README.md#how-do-i-disable-the-ingress-controller)
-2. SSH into the GCE master node and delete the GLBC manifest file found at `/etc/kubernetes/manifests/glbc.manifest`
-3. Create the GKE cluster without it:
+As of Kubernetes 1.3, GLBC runs as a static pod on the master.
+If you want to disable it, you have 3 options:
+
+### Soft disable
+
+1. Have it no-op for an Ingress resource based on the `ingress.class` annotation as shown [here](README.md#how-do-i-disable-an-ingress-controller).
+This can also be used to use one of the other Ingress controllers at the same time as the GCE controller.
+
+### Hard disable
+
+2. SSH into the GCE master node and delete the GLBC manifest file found at `/etc/kubernetes/manifests/glbc.manifest`.
+3. Disable the addon in GKE:
```

Comment:
I think this isn't supposed to be here

Reply:
Can you elaborate? The goal is to have one heading about disabling the add-on instead of 4 separate places to update.